src/components/courses/AdminPage.tsx

```
@@ -0,0 +1,391 @@
+                  }}
+                  className="text-blue-500 dark:text-blue-400 hover:text-blue-700"
+                >
+                  <FaEdit />
+                </button>
+                <button
+                  onClick={() => courseApi.delete(course.id)}
+                  className="text-red-500 dark:text-red-400 hover:text-red-700"
+                >
+                  <FaTrash />
+                </button>
+              </div>
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)